### ISSUE TITLE
As a Developer I would like to move to python 3 so we avoid technical risks in the future

### ISSUE THREAD

**Issue by jimenofonseca** [architecture-building-systems/CityEnergyAnalyst#2578]

**Comment by shanshanhsieh** [comment]
@reyery could you please make a new issue for the testing plan before we start looking into it?

**Comment by daren-thomas** [comment]
@shanshanhsieh for testing, we expect to do the following:

- implement coverage of (most) of the code base (we'll have to remove "dead" code)
- run the test workflow "slow" and collect the results
- write a script to compare those results with a new run
- this will give a yay / nay output on whether we're still running the "same" CEA
